internal/serviceapi/logintegration/resource_test.go

```diff
@@ -0,0 +1,283 @@
+package pushbasedlogexportv3api_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+const (
+	resourceName       = "mongodbatlas_push_based_log_export_v3_api.test"
+	nonEmptyPrefixPath = "push-log-prefix-v3"
+)
+
+func TestAccPushBasedLogExportV3API_basic(t *testing.T) {
+	resource.Test(t, *basicTestCase(t))
```

Comment:
```suggestion
	resource.ParallelTest(t, *basicTestCase(t))
```
I believe we can run safely in parallel, can you confirm if API already supports creating multiple log integrations for a same project?

Reply:
you are right, tests with different log_integration types can be run in parallel. API supports creating multiple log integrations in the same project if they have different types, so once we have more types supported, we will be able to have more than one log_integration per project. For now, limit per project is 1 log_integration because we only have one type: S3_LOG_EXPORT

Reply:
The error when trying to create a second log_integration with the same type is very clear:
```
POST: HTTP 400 Bad Request (Error code: "INTEGRATION_LIMIT_REACHED") Detail: Maximum number of S3_LOG_EXPORT integration configurations reached for this
│ project. Reason: Bad Request. Params: [S3_LOG_EXPORT], BadRequestDetail: 
```

Reply:
Thanks for confirming 👍 Just as FYI in longer term plan the team is also planning to support multiple S3 log integrations configurations in a single project (e.g. exporting logs of a project to 2 different S3 buckets)